asynch/proto/utils/escape.py

```diff
@@ -47,10 +49,16 @@ def escape_param(item):
         return item
 
 
-def escape_params(params):
-    escaped = {}
+def escape_params(params: Mapping[str, Any]) -> MappingProxyType[str, str]:
```

Comment:
I mean returning `dict` may be OK, but I am not insisting, just pointing out the idea that if returning a dictionary is simpler and consistent with the code base, then returning `dict` seems to be good (at least so far).